sys/src/main/java/com/alibaba/nacos/sys/env/EnvUtil.java

```diff
@@ -36,159 +39,182 @@
 import java.io.Reader;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 /**
  * Its own configuration information manipulation tool class.
  *
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
 public class EnvUtil {
-    
+
```

Comment:
please use nacos code style to reformat codes

Reply:
ok👌